model/channel.go

```diff
@@ -27,6 +27,7 @@ type Channel struct {
 	Description   string `json:"description"`
 	LastPostAt    int64  `json:"last_post_at"`
 	TotalMsgCount int64  `json:"total_msg_count"`
+	ExtraUpdateAt int64  `json:"extra_update_at"`
```

Comment:
If we increment Channel.UpdateAt then that invalidates the channel as well and that information has to be pulled down again as well. Since we bother to have them as separate API calls, I figured we wanted them to have separate etags.